packages/enketo-transformer/src/xsl/openrosa2html5form.xsl

```diff
@@ -254,9 +257,12 @@ XSLT Stylesheet that transforms OpenRosa style (X)Forms into valid HTMl5 forms
                     <xsl:if test="//xf:submission">
                         <xsl:message>ERROR: Submissions element(s) not supported.</xsl:message>
                     </xsl:if>
+                    <xsl:comment>End of form</xsl:comment>
```

Comment:
How is this comment and the big diff for snapshots related to the background audio? 🤔 IMHO would be nicier to PR separately.